docs/api/version-history.md

```diff
@@ -64,6 +64,7 @@ keywords: "API, Docker, rcli, REST, documentation"
   to return those without the specified labels.
 * `POST /containers/create`, `GET /containers/{id}/json`, and `GET /containers/json` now supports
   `BindOptions.NonRecursive`.
+* `POST /services/create` and `POST /services/{id}/update` now supports `BindOptions.NonRecursive`.
```

Comment:
double checking; the swagger doesn't need updating? This is the type that's already defined here? https://github.com/moby/moby/blob/6f8c671d702197a189d162d86a3f4cccfa5a3db2/api/swagger.yaml#L305-L308

Reply:
It was already defined, but not implemented and not documented in this markdown.